tests/test_validators.py

```diff
@@ -72,6 +74,22 @@ def test_public_email_domain(self):
             self.assertIn('webmail_domain', self.webmail_form.errors)
             self.assertNotIn('not_webmail_domain', self.webmail_form.errors)
 
+    def test_public_email_domain_helper(self):
+        with self.app.test_request_context('/'):
+            self.assertEqual(is_public_email_domain(u'gmail.com', default=False), True)
+            self.assertEqual(is_public_email_domain(u'google.com', default=False), False)
+            # this domain lookup fails, so the helper should return default value
+            self.assertEqual(
+                is_public_email_domain(u'www.abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijks.com', default=False),
+                False)
+            self.assertEqual(
+                is_public_email_domain(u'www.abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijks.com', default=True),
+                True)
+            # as no default is provided and the domain will trigger MXLookupExceptiom
+            # the helper function will raise as expection
+            with self.assertRaises(MXLookupException):
+                is_public_email_domain(u'www.abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijks.com')
```

Comment:
Place this test above the tests with the default, so that it's obvious what the default is doing there.